docs/modules/ROOT/pages/getting-started/assembly-registry-reference.adoc

```diff
@@ -6,16 +6,18 @@ include::{mod-loc}shared/all-attributes.adoc[]
 = {registry} reference
 //If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
 
-This chapter lists the supported artifact types and content rule types that are stored in {registry}. 
-
+This chapter lists the supported artifact types, states and content rule types that are stored in {registry}.
+transitions
```

Comment:
Just the word "transitions" all by itself?

Reply:
Copy-paste mistake, I guess.